### PR TITLE
fix(ui): airdrop gem tracker ui changes

### PR DESCRIPTION
**Pull request by peps**
Various copy and UI tweaks to the AirdropGiftTracker based on internal feedback. 

CleanShot 2024-10-01 at 15 35 22

CleanShot 2024-10-01 at 15 36 55

CleanShot 2024-10-01 at 15 37 34

### CODE REVIEW

File: src/containers/Airdrop/AirdropGiftTracker/components/Claimmodal/ClaimModal.tsx [hunk 1]
```diff
@@ -82,9 +93,14 @@ export default function ClaimModal({ onSubmit, onClose }: ClaimModalProps) {
                             <XLogoIcon />
                         </XLogo>
                     </ClaimButton>
-                </ActionWrapper>
-                <ActionWrapper>
-                    <TextButton onClick={onClose}>{t('doLater')}</TextButton>
+
+                    {!allowTelemetry && (
+                        <FinePrint
+                            dangerouslySetInnerHTML={{
```
Comment (shanimal08) [inline src/containers/Airdrop/AirdropGiftTracker/components/Claimmodal/ClaimModal.tsx:99]: can we use the `Trans` component for this? :D _dangerously_ always makes me nervous:p

File: src/containers/Airdrop/AirdropGiftTracker/sections/LoggedIn/segments/Invite/Invite.tsx [hunk 2]
```diff
@@ -84,7 +65,7 @@ export default function Invite() {
                 </GemPill>
             </InviteButton>
 
-            {nextBonusTier && (
+            {/*nextBonusTier && (
```
Comment (shanimal08) [inline src/containers/Airdrop/AirdropGiftTracker/sections/LoggedIn/segments/Invite/Invite.tsx:68]: i'd personally just remove the commented out code entirely

**Review by shanimal08 — APPROVED** [review]
some nits but LGTM :D haven't tested locally yet though because i'm running the build atm _trying_ to win something 😿

File: src/containers/Airdrop/AirdropGiftTracker/components/Claimmodal/ClaimModal.tsx (hunk 1 above)
Comment (peps) [reply src/containers/Airdrop/AirdropGiftTracker/components/Claimmodal/ClaimModal.tsx]: too dangerous! done!

File: src/containers/Airdrop/AirdropGiftTracker/sections/LoggedIn/segments/Invite/Invite.tsx (hunk 2 above)
Comment (peps) [reply src/containers/Airdrop/AirdropGiftTracker/sections/LoggedIn/segments/Invite/Invite.tsx]: done

**Comment by Anziskavds** [comment]
@peps The only thing a found that's different is on the below screen on the document(https://docs.google.com/document/d/1IMA6KPQtGJ82IqEv-UU8TAv2__duZFA-1-DifgVtU-s/edit) it says "Check your email" but on Tari Universe/Airdrop it show's "Check your inbox". Not sure if this was changed and that I missed it. 
image

**Comment by Anziskavds** [comment]
Tested the UI changes on Prod and everything passed as per the document here: https://docs.google.com/document/d/1IMA6KPQtGJ82IqEv-UU8TAv2__duZFA-1-DifgVtU-s/edit

I couldn't test the "The SEC DIsclaimer page" I see it's only for US users.

cc @peps @stringhandler

**Pull request merged**